tests/integration/aliases/step1/index.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
as annoying as it might be.  i'd recommend that we make separate tests for all of these.  just to make it very easy and clear to understand what's going on.

Reply:
Good idea - went ahead and did this.